### ISSUE TITLE
Opening a Undetermined toolbox in a storage implant deletes some items

### ISSUE THREAD

**Issue by SphiraI** [space-wizards/space-station-14#24831]
## Description
If you were to roll Syndie *and* thief, and get a storage implanter via uplink, place your undetermined toolbox in said implant, then proceed to select and open the toolbox, some items *might* get added to the implant, others might just get deleted. And sometimes the items just drop on the ground as intended?

**Reproduction**
- Be thief
- Obtain storage implanter via non-thief ways
- Place undetermined toolbox in storage implant
- Use the toolbox *inside* the implant and approve it
- Cry at lost items

**Additional context**
With a little bit of testing, the issue only seems to happen when the toolbox tries to deposit the items into a storage that would be able to hold one kit, but not both. eg. Chem kit and Syndie kit. Selecting both in the implanter will deposit the chem kit, but delete the syndie kit.

**Comment by ThatGuyUSA** [comment]
Yes, this is still like this. However the undetermined thieving toolbox has since been replaced with an undetermined thieving satchel which does not suffer from this problem in normal play.

I am unsure why both exist, the satchel merely parents from the toolbox to function.

https://github.com/user-attachments/assets/c91f31d3-c57f-4936-9189-7af325516f33

https://github.com/user-attachments/assets/2f8f85bb-af7e-47b1-ab9e-bf13e05403da
